edb/edgeql/parser/parser.py

```diff
@@ -1,193 +0,0 @@
-#
-# This source file is part of the EdgeDB open source project.
-#
-# Copyright 2008-present MagicStack Inc. and the EdgeDB authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-from __future__ import annotations
-
-from typing import *
-
-from edb import errors
-from edb.common import parsing
-
-from .. import tokenizer
-
-import edb._edgeql_parser as ql_parser
-
-
-class EdgeQLParserSpec(parsing.ParserSpec):
-    def get_parser(self):
-        return EdgeQLParser(self)
-
-
-class EdgeQLSingleSpec(EdgeQLParserSpec):
-    def get_parser_spec_module(self):
-        from .grammar import single
-
-        return single
```

Comment:
I missed an unnecessary class, that I could have removed in favor of a single module reference!